models/attack.go

```diff
@@ -0,0 +1,23 @@
+package models
+
+import "errors"
```

Comment:
`github.com/pkg/errors` is what we have as an import in other files. Maybe we should use one of them uniformly? thoughts?